packages/yew/src/virtual_dom/vtag.rs

```diff
@@ -92,6 +92,19 @@ impl InputFields {
     }
 }
 
+// Clippy doesn't recognise that `defaultvalue` is read in other files and in this very file when
```

Comment:
Should we change the way we run clippy on the codebase for it to parse the code in "ssr" and "csr" features as well? I think this will remove the need to use these escape hatches

Reply:
We already do this in CI. Our CI iterates all combinations of features and lints them one by one.

See: https://github.com/yewstack/yew/blob/master/ci/feature-soundness.sh

Clippy is "correct" on this issue. This field is only read if feature ssr or csr is enabled.
The "proper" way clippy ways wants you to do is to apply feature gate on this field so it only presents when feature csr or ssr is enabled.

However, we can assume that we have more knowledge than clippy since we can assume end users always want either csr or ssr enabled. We can suppress this lint here.